src/imex.rs

```diff
@@ -107,6 +107,7 @@ pub async fn imex(
     } else {
         info!(context, "IMEX successfully completed");
         context.emit_event(EventType::ImexProgress(1000));
+        context.emit_event(EventType::AccountsItemChanged);
```

Comment:
Isn't it sufficient to have emitting of this event in `get_backup2()`? `imex()` also imports keys, not only backups, and also exports backups.

Reply:
`get_backup2()` would not be sufficient as this is not about networked import, see https://github.com/deltachat/deltachat-desktop/issues/4250

Though we can talk about the key import/export and the backup export, you are right that the events are not needed there, only on backup import. But an extra event there won't hurt either, so not a big deal in this case, though we can still wrap t in an if statement to only trigger on backup import

Reply:
I changed this to:
```rs
if what == ImexMode::ImportBackup {
    context.emit_event(EventType::AccountsItemChanged);
}
```

Reply:
`import_backup_stream()` is the correct place as far as i see, in order to not generate extra events and not duplicate the code.